src/writer.js

```diff
@@ -18,6 +18,17 @@ class Writer {
     return this._queue.length
   }
 
+  printableJsonTrace (formattedTrace) {
+    return JSON.stringify(formattedTrace.map(trace => {
```

Comment:
Technically it's a span and not a trace. And yes, I know I did the same mistake in the same file 😅